ts/src/gemini.ts

```diff
@@ -78,7 +78,6 @@ export default class gemini extends Exchange {
                 'fetchTradingFee': false,
                 'fetchTradingFees': true,
                 'fetchTransactions': 'emulated',
-                'postOnly': true,
```

Comment:
this should be replaced with `createPostOnlyOrder`

Reply:
@carlosmiei Thanks for pointing me in the right direction.  I see how that uses the base Exchange.ts, and have updated gemini.ts to use createPostOnlyOrder.